iChef-WebUi/src/app/components/landing-page/landing-page.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
+import { PATHS } from 'src/app/constants/paths';
+import { BUTTON, LANDING_PAGE, PLACEHOLDERS } from 'src/app/constants/texts';
+
+@Component({
+  selector: 'app-landing-page',
+  templateUrl: './landing-page.component.html',
+  styleUrls: ['./landing-page.component.scss']
+})
+export class LandingPageComponent {
+
+  readonly landingPageImage = PATHS.LANDING_PAGE_IMAGE;
+  readonly appTitle = LANDING_PAGE.TITLE;
```

Comment:
The names of the constant classes are not explicit enough. If we really want to use them I would rename them to BUTTON_TEXTS or something, but I feel like we don't need them since we are not going to localize those strings anyway. We should just use plain strings in html and if we decide that we need to set some constants for the string we will do that.

Reply:
The BUTTON."TEXTS" are used in more than one place and it would be nice to have all the descriptions in one place for similar components. If you we still think that we don't need the constants, they can be removed. Tho in my opinion it's more organized this way (with the BUTTON.<which_button>).

Reply:
As I said, if you feel that this is more organized then continue this way, but make the names more suggestive because as I was reading the page I thought that they were references to components. Add them to a class "Strings" or group them somehow such that the name of the variable is states clearly what that variable holds. Ex:
- Strings.LandingPage.Title
- LandingPageStrings.Title
- LandingPage.TitleString